apis/python/node/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+use dora_node_api::config::DataId;
+use dora_node_api::{DoraNode, Input};
+use eyre::Context;
+use futures::StreamExt;
+use pyo3::prelude::*;
+use pyo3::types::PyBytes;
+use std::sync::Arc;
+use std::thread;
+use tokio::sync::mpsc;
+use tokio::sync::mpsc::{Receiver, Sender};
+#[pyclass]
+// #[repr(transparent)]
+pub struct PyDoraNode {
+    // pub node: DoraNode,
+    pub rx_input: Receiver<Input>,
+    pub tx_output: Sender<(String, Vec<u8>)>,
+}
+
+pub struct PyInput(Input);
+
+impl IntoPy<PyObject> for PyInput {
+    fn into_py(self, py: Python) -> PyObject {
+        (self.0.id.to_string(), PyBytes::new(py, &self.0.data)).into_py(py)
+    }
+}
+
+#[pymethods]
+impl PyDoraNode {
+    #[staticmethod]
+    pub fn init_from_env() -> Self {
+        let (tx_input, rx_input) = mpsc::channel(10);
+        let (tx_output, mut rx_output) = mpsc::channel::<(String, Vec<u8>)>(10);
```

Comment:
I think we should set the channel capacity to the minimum here:

```suggestion
        let (tx_input, rx_input) = mpsc::channel(1);
        let (tx_output, mut rx_output) = mpsc::channel::<(String, Vec<u8>)>(1);
```

This way we reduce the delay between the return of `send_output`/`next` and the actual send/receive operation.